nvflare/app_common/executors/splitnn_learner_executor.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pt.learners.cifar10_learner_splitnn import SplitNNConstants
```

Comment:
These constants need to be defined in nvflare/app_common so we can import them here.

Reply:
Good catch. This was the wrong import. It should be `from nvflare.app_common.workflows.splitnn_workflow import SplitNNConstants`. Fixed.